airbyte-integrations/connectors/source-klaviyo/integration_tests/abnormal_state.json

```diff
@@ -2,67 +2,95 @@
   {
     "type": "STREAM",
     "stream": {
-      "stream_state": { "datetime": "2120-10-10T00:00:00Z" },
-      "stream_descriptor": { "name": "events" }
+      "stream_state": {
+        "datetime": "2120-10-10 00:00:00+00:00"
+      },
+      "stream_descriptor": {
+        "name": "events"
+      }
     }
   },
   {
     "type": "STREAM",
     "stream": {
-      "stream_state": { "updated": "2120-10-10T00:00:00Z" },
-      "stream_descriptor": { "name": "global_exclusions" }
+      "stream_state": {
+        "updated": "2120-10-10T00:00:00+00:00"
+      },
+      "stream_descriptor": {
+        "name": "global_exclusions"
+      }
     }
   },
   {
     "type": "STREAM",
     "stream": {
       "stream_state": {
-        "updated": "2120-10-10 00:00:00",
+        "updated": "2120-10-10T00:00:00+00:00",
         "archived": {
-          "updated": "2120-10-10 00:00:00"
+          "updated": "2120-10-10T00:00:00+00:00"
         }
       },
-      "stream_descriptor": { "name": "flows" }
+      "stream_descriptor": {
+        "name": "flows"
+      }
     }
   },
   {
     "type": "STREAM",
     "stream": {
-      "stream_state": { "updated": "2120-10-10 00:00:00" },
-      "stream_descriptor": { "name": "metrics" }
+      "stream_state": {
+        "updated": "2120-10-10T00:00:00+00:00"
+      },
+      "stream_descriptor": {
+        "name": "metrics"
+      }
     }
   },
   {
     "type": "STREAM",
     "stream": {
-      "stream_state": { "updated": "2120-10-10 00:00:00" },
-      "stream_descriptor": { "name": "lists" }
+      "stream_state": {
+        "updated": "2120-10-10T00:00:00+00:00"
+      },
+      "stream_descriptor": {
+        "name": "lists"
+      }
     }
   },
   {
     "type": "STREAM",
     "stream": {
       "stream_state": {
-        "updated_at": "2120-10-10 00:00:00",
+        "updated_at": "2120-10-10T00:00:00+00:00",
```

Comment:
Why we need update state format?

Reply:
I updated these so the formats match what is returned from the API